### DISCUSSION TITLE
Setting up magma for the first time

### DISCUSSION THREAD

**Discussion by mimo123-code** [magma/magma#5557] [Q&A]
Hello  I am new to magma and i am unable to follow the quick guide . can anybody help me out in the installation

**Comment by ShubhamTatvamasi** [comment]
Were you able to build Orchestrator?

**Reply by mimo123-code** [reply]
yes i was able to build the orchestrator. i used the command ./build.py -a in the quick guide

**Reply by ShubhamTatvamasi** [reply]
Is it up and running?

**Reply by mimo123-code** [reply]
when i put the password in nms ui . i am unable to enter the page . It says page isn't working redirected too many times

**Reply by karthiksubraveti** [reply]
@mimo123-code Did you run through 
https://docs.magmacore.org/docs/basics/quick_start_guide#using-the-nms-ui

`HOST [magma/nms/app/packages/magmalte] $ ./scripts/dev_setup.sh` 
?
can you also dump magmalte and orc8r controller logs?

**Reply by hcgatewood** [reply]
@mimo123-code what version of the Orchestrator are you building? There was a too-many-redirects issue a couple versions ago

**Reply by mimo123-code** [reply]
I am following the quick start guide for installing the orchestrator and nms ui .. is it not the page I need to follow??

**Reply by mimo123-code** [reply]
> @mimo123-code Did you run through
> https://docs.magmacore.org/docs/basics/quick_start_guide#using-the-nms-ui
> 
> `HOST [magma/nms/app/packages/magmalte] $ ./scripts/dev_setup.sh`
> ?
> can you also dump magmalte and orc8r controller logs?

I have used the particular commands only

**Reply by mimo123-code** [reply]
Now I am getting gateway error..

**Comment by karthiksubraveti** [comment]
@mimo123-code Can you provide more details here on what gateway error you are seeing ? Dump the logs may be.

**Reply by mimo123-code** [reply]
i am getting gateway error 502 . till now i have only setup  the orchestrator part .  ./scripts/dev_setup.sh
 command is also running fyn. from where can i get the logs file

**Reply by karthiksubraveti** [reply]
Is access to NMS fixed ? Also can you share magmalte and orc8r controller container logs. Will provide more insight into what might be going on here. Also where are you seeing gateway error ? Are you trying to register the gateway to Orc8r ?

**Reply by karthiksubraveti** [reply]
https://docs.docker.com/engine/reference/commandline/logs/

**Reply by mimo123-code** [reply]
hello Everybody i am getting error while running the command "make run" during setting up access gateway .

OAI_FLAGS -DS6A_OVER_GRPC=True -DEMBEDDED_SGW=True -DENABLE_OPENFLOW=True -DSPGW_ENABLE_SESSIOND_AND_MOBILITYD=True
sudo service magma@* stop
make -C /lte/gateway/python buildenv
make[1]: *** /lte/gateway/python: No such file or directory.  Stop.
Makefile:136: recipe for target 'build_python' failed
make: *** [build_python] Error 2
